A F&B manager wants to determine whether there is any significant difference in the diameter of the cutlet between two units. A randomly selected sample of cutlets was collected from both units and measured? Analyze the data and draw inferences at 5% significance level. Please state the assumptions and tests that you carried out to check validity of the assumptions.
<br>
Minitab File : Cutlets.mtw
<br>
Solution:-
<br>
H0 μ1 = μ2 (There is no significant difference in the diameter of the cutlet between two units).
<br>
Ha μ1 /= μ2 (There is significant difference in the diameter of the cutlet between two units).
<br>
So, Two sample two tail test is applicable.

In [ ]:
from scipy import stats as st
import numpy as np
import pandas as pd

In [4]:
cutlets = pd.read_csv("Cutlets.csv")

In [5]:
cutlets.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

In [34]:
Unit_A = cutlets.iloc[:,0]
Unit_B = cutlets.iloc[:,1]

In [35]:
print("P_value :-",np.round(st.ttest_ind(Unit_A,Unit_B)[1],3))

P_value :- 0.472


Here,  P_value = 0.472
<br>
P_value > 0.05 --> Fail to Reject H0 ………………………………For 5% significance level.
<br>
Hence, there no significant difference in the diameter of the cutlet between two units A and B.